src/java.base/share/classes/java/nio/file/Files.java

```diff
@@ -1719,6 +1719,9 @@ private static List<FileTypeDetector> loadInstalledDetectors() {
      * @throws  SecurityException
      *          If a security manager is installed and it denies an unspecified
      *          permission required by a file type detector implementation.
+     *
+     * @spec https://www.rfc-editor.org/info/rfc2045
+     *      RFC 2045: RFC 2045: Multipurpose Internet Mail Extensions (MIME) Part One: Format of Internet Message Bodies
```

Comment:
Maybe this one can be put on two lines to avoid the wrapping when looking at is side-by-side.

Reply:
Fixed. There was also a stutter-bug with a double "RFC 2045: RFC 2045:" which I have also fixed.